features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml

```diff
@@ -252,4 +252,10 @@
     -->
     <!--<AllowCustomClaimMappingsForAuthenticators>true</AllowCustomClaimMappingsForAuthenticators>-->
 
+    <!--
+    Allow federated authenticators to use IDP claim mappings combined with the default authenticator claim mappings.
+    Default value: false
+    -->
+    <!--<MergeCustomClaimMappingsWithDefaultForAuthenticators>true</MergeCustomClaimMappingsWithDefaultForAuthenticators>-->
```

Comment:
Update the comment with `AllowMergingCustomClaimMappingsWithDefaultClaimMappings `